package.json

```diff
@@ -7,16 +7,22 @@
     "@emotion/styled": "^11.10.4",
     "@mui/icons-material": "^5.10.9",
     "@mui/material": "^5.10.10",
+    "@reduxjs/toolkit": "^1.8.6",
```

Comment:
- [ ] Kindly deploy your app and add a link to it to your readme.  Your app is really beautiful.  Let's show it off to the world! (and potential employers, of course).

That way, your users won't have to install your app locally in order to check it out, and potential employers will be more likely to take a look. 💪